maro/rl/learner/abs_learner.py

```diff
@@ -6,14 +6,16 @@
 
 class AbsLearner(ABC):
     def __init__(self):
+        """Abstract learner class to control the policy learning process...
+        """
         pass
```

Comment:
insert line 9 between line 7 and line 8 (as the summary of this class). Also, push the double """ in one line if not too long.